package.json

```diff
@@ -20,5 +20,15 @@
   "homepage": "https://github.com/TBDG/chatter#readme",
   "devDependencies": {
     "concurrently": "^5.3.0"
+  },
```

Comment:
This is better to be installed in the public or server directory not in the root